### PR TITLE
Meta tags + simple favicon

### PR DESCRIPTION
**Pull request by themrb**
I've set up the meta tags and open graph tags. The text and the thumb image could use some refining,  this is just what I came up with quickly. They also need to be tested, possibly on the staging site. (https://developers.facebook.com/tools/debug/)

I've also whipped up a very simple favicon and set it up.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,15 @@
...
+    <meta name="description" content="This website takes publically available data from the Australian Electoral Commission (AEC) Annual Returns Locator Service, and converts it to a form that can easily be visualised and analysed.">
     <meta name="author" content="">
-    <link rel="shortcut icon" href="../../assets/ico/favicon.png">
+    <link rel="shortcut icon" href="img/favicon.png">
+
+    <meta property="og:url"                content="http://au.politicaldonations.info" />
+    <meta property="og:type"               content="website" />
+    <meta property="og:title"              content="Australian Political Donations Visualiser" />
+    <meta property="og:description"        content="Explore the money behind the Australian political parties." />
+    <meta property="og:image"              content="img/thumb.png" />
```
Comment (river-jade) [inline index.html]: just one space between html tag properties pls.

**Comment by river-jade** [comment]
Great, thanks for the super speedy turnaround Jarrah! I've taken another screenshot that I think looks better. https://drive.google.com/open?id=0B1ldWXmfbtLEZWlHUFNMdDRXalU

**Comment by themrb** [comment]
I thought a 'whole' image would be too small when turned into a thumb image on Facebook, but I was wrong - it works quite well https://developers.facebook.com/tools/debug/og/object/?q=http%3A%2F%2F114.198.116.71%3A8001%2F

**Comment by river-jade** [comment]
Great, LGTM. Yeah, git rebase can be very tricky. Pro tip: before doing anything scary in git, create a branch at current head. eg git branch tmp. git reflog is also your friend for when things get seriously borked, but putting a branch at the tip is a lifesaver. Reading the link I sent will help to understand why.

**Pull request merged**